diff --git a/lib/authorization/stateful_roles.rb b/lib/authorization/stateful_roles.rb
index 773d6dd..437605a 100644
--- a/lib/authorization/stateful_roles.rb
+++ b/lib/authorization/stateful_roles.rb
@@ -9,7 +9,7 @@ module Authorization
       recipient.class_eval do
         include StatefulRolesInstanceMethods
         
-        acts_as_state_machine :initial => :pending
+        acts_as_state_machine :initial => :active
         state :passive
         state :pending, :enter => :make_activation_code
         state :active,  :enter => :do_activate
@@ -17,7 +17,7 @@ module Authorization
         state :deleted, :enter => :do_delete
 
         event :register do
-          transitions :from => :passive, :to => :pending, :guard => Proc.new {|u| !(u.crypted_password.blank? && u.password.blank?) }
+          transitions :from => :passive, :to => :active, :guard => Proc.new {|u| !(u.crypted_password.blank? && u.password.blank?) }
         end
         
         event :activate do
