diff --git a/lib/prawnto/template_handler/compile_support.rb b/lib/prawnto/template_handler/compile_support.rb
index 3a45b8b..cb27660 100644
--- a/lib/prawnto/template_handler/compile_support.rb
+++ b/lib/prawnto/template_handler/compile_support.rb
@@ -31,7 +31,7 @@ module Prawnto
 
       # added to make ie happy with ssl pdf's (per naisayer)
       def ssl_request?
-        @controller.request.env['SERVER_PROTOCOL'].downcase == "https"
+        @controller.request.env['SERVER_PROTOCOL'].try(:downcase) == "https"
       end
       memoize :ssl_request?
 
